src/settings/Passwd.js

```diff
@@ -95,7 +96,16 @@ class Passwd extends React.Component {
           throw new Error(intl.formatMessage({ id: 'ui-developer.passwd.error.missingUser' }, { username: values.username }));
         }
       })
-      .then(() => mutator.isLocalPasswordSet.GET({ params: { userId } }))
+      .then(async () => {
+        try {
+          const response = await mutator.isLocalPasswordSet.GET({ params: { userId } });
+          return response;
+        } catch (e) {
+          return {
+            credentialsExist: false,
+          };
+        }
+      })
```

Comment:
Instead of mixing a promise chain and async/await, it would be better to convert the whole thing to async/await.

Reply:
Since this is a CSP, I didn't want to make too many changes.